python/nano/src/setup.py

```diff
@@ -104,7 +105,9 @@ def setup_package(plat_name):
         url='https://github.com/intel-analytics/BigDL',
         install_requires=install_requires,
         extras_require={"tensorflow": tensorflow_requires,
-                        "pytorch": pytorch_requires},
+                        "pytorch": pytorch_requires,
+                        "inc": ["neural_compressor==1.8.1"]
+                        },
```

Comment:
@yangw1234  How about this installation option? We can install by :
```shell
pip install bigdl-nano.whl[pytorch, inc]
```
I think we can also put a `ipex` option to install ipex at the same time.

Reply:
I think we had a ipex installation setup but removed for some pypi safety policy?